scalyr_agent/agent_main.py

```diff
@@ -404,6 +413,11 @@ def __perform_config_checks(self, no_check_remote):
                                         'Please visit https://www.scalyr.com/keys and copy a Write Logs key into the '
                                         '\'api_key\' field in the configuration file' % self.__config.file_path)
                     else:
+                        log.error('Failed to send request to the server.  The server address could be '
+                                  'wrong, there maybe a network connectivity issue, or the provided '
```

Comment:
Let's rewrite that second and subsequent sentences to be "The server address could be wrong, there could be a network connectivity issue, or the provided_token could be incorrect.  The agent will keep trying to connect in the background.  You can disable this check with --no-check-remote-server."

Reply:
Where you wrote "provided_token" did you mean that to be a space, like "provided token"?